public/js/angular/controllers.js

```diff
@@ -254,44 +246,42 @@ angular
       $scope.weblit = weblit;
 
       $scope.wlcPoints = CONFIG.wlcPoints;
-
     }
   ])
-  .controller('competencyMediaGenController', ['$scope', '$sce',
+  .controller("competencyMediaGenController", ["$scope", "$sce",
```

Comment:
I always thought that way was clever, but not as readable. Which was my only reason for picking the if/else over anything else.